test/integration/fixtures/blocks/core__comments__deprecated-1.json

```diff
@@ -4,6 +4,7 @@
 		"isValid": true,
 		"attributes": {
 			"tagName": "div",
+			"legacy": false,
```

Comment:
As noted in the PR description, maybe we should remove all cases where we set a default of `false` for boolean types.